src/Twig/Func/ResolveIpFunction.php

```diff
@@ -58,7 +58,7 @@ public function __invoke(string $hostname = null): ?string
             return null;
         }
 
-        return reset($ipAddresses);
+        return (false !== reset($ipAddresses)) ? reset($ipAddresses) : null;
```

Comment:
We should either write:

```php
return reset($ipAddresses) ?: null;
```

Or, probably better: Include a check in the early return above:

```php
if (false === $ipAddresses || [] === $ipAddresses) {
    return null;
}

return reset($ipAddresses);
```

Reply:
Don't like my style, ha? No, problem. However, your first suggestion is not allowed with `declare(strict_types=1);`. I shall hence write it more comprehensively.